recipe/bld.bat

```diff
@@ -20,10 +20,9 @@ git clone https://github.com/CGAL/cgal-swig-bindings.git csb
 cd csb
```

Comment:
For reproducibility and for information purposes, it's better to use a git commit rather than the git master and then update the git commit when needed.